### PR TITLE
chore: remove chopsticks

### PR DESCRIPTION
**Pull request by bonomat**
One container less to start 🥳
Instead we will need to ensure the wallet is loaded and funded.

### CODE REVIEW

**Review by holzeis — APPROVED** [review]
Thanks a lot for that! 🥳

File: crates/tests-e2e/examples/fund.rs [hunk 1]
```diff
@@ -39,11 +51,55 @@ pub struct Opts {
...
+            tracing::info!("Wallet not loaded in Bitcoind, loading it... {err:#}");
+
+            rpc.load_wallet("regtest")
+                .expect("To be able to load wallet");
+        }
+    }
+
+    let i = rpc.get_block_count()?;
+    tracing::info!("Bitcoin has {i} blocks");
+    if i < 101 {
```
[the remark this answers is not in the document]
Comment (bonomat) [reply crates/tests-e2e/examples/fund.rs:91]: `100 < 101 = true` no? So we would mine 1 block.

[the remark this answers is not in the document]
Comment (holzeis) [reply crates/tests-e2e/examples/fund.rs:91]: 🙈 it's too late for me to comment on PRs 😅

**Comment by bonomat** [comment]
Looks like the e2e setup for CI needs some more work. Not putting more work into this now as it's not that important.

**Pull request closed without merging**